packages/dd-trace/src/plugins/util/git.js

```diff
@@ -47,16 +48,20 @@ function getCommitsToUpload (commitsToExclude) {
 function generatePackFilesForCommits (commitsToUpload) {
   const tmpFolder = os.tmpdir()
 
-  const prefix = Math.floor(Math.random() * 10000)
-  const path = `${tmpFolder}/${prefix}`
+  const randomPrefix = Math.floor(Math.random() * 10000)
+  const temporaryPath = path.join(tmpFolder, randomPrefix)
 
-  const orderedCommits =
-    execSync(
-      `git pack-objects --compression=9 --max-pack-size=3m ${path}`,
-      { input: commitsToUpload.join('\n') }
-    ).toString().split('\n').filter(commit => !!commit)
+  try {
+    const orderedCommits =
+      execSync(
+        `git pack-objects --compression=9 --max-pack-size=3m ${temporaryPath}`,
+        { input: commitsToUpload.join('\n') }
+      ).toString().split('\n').filter(commit => !!commit)
```

Comment:
same as the other PR
```suggestion
      ).toString().split('\n').filter(commit => commit)
```